paper-tabs.html

```diff
@@ -364,24 +368,43 @@
       );
     },
 
-    _scroll: function() {
-      var scrollLeft;
 
+    _scroll: function(e, detail) {
       if (!this.scrollable) {
         return;
       }
 
-      scrollLeft = this.$.tabsContainer.scrollLeft;
+      var ddx = (detail && -detail.ddx) || 0;
+      this._affectScroll(ddx);
+    },
+
+    _down: function(e) {
+      // go one beat async to defeat IronMenuBehavior
+      // autorefocus-on-no-selection timeout
+      this.async(function() {
+        if (this._defaultFocusAsync) {
```

Comment:
`_defaultFocusAsync` is used in IronMenuBehavior: https://github.com/PolymerElements/iron-menu-behavior/blob/master/iron-menu-behavior.html#L244-L255